src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/uploaddocuments/UploadDocumentFurtherEvidenceHandler.java

```diff
@@ -53,9 +55,12 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
             response.addError("You need to upload PDF documents only");
             return response;
         }
+
         moveDraftsToSscsDocs(caseData);
         initDraftSscsFurtherEvidenceDocument(caseData);
+        caseData.setEvidenceHandled("No");
```

Comment:
Are we setting always no, or is there any condition to check?

Reply:
any failure conditions have early returned by this point in the code. EvidenceHandled flag applys to all cases for which this event is successfully applied